app/src/webview/java/org/mozilla/focus/web/WebViewProvider.java

```diff
@@ -142,6 +142,17 @@ public static void disableBlocking(WebSettings settings, WebView webView) {
         CookieManager.getInstance().setAcceptCookie(true);
     }
 
+    public static void requestDesktopSite(WebSettings settings) {
+        settings.setUserAgentString("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Safari/537.36");
```

Comment:
It would be nice if we can construct the desktop UA from the WebView UA like we do for the default UA. At least the Chrome version would be good to keep in sync so that we do not confuse scripts that try to detect features based on the version.

@miketaylr may have some input too!